useful-scripts/misc-test-utils/src/services/releaseService.ts

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-console */
+import axios from 'axios';
+import chalk from 'chalk';
+import errorHandler from '../utils/errorHandler';
+
+const { ADMIN_URL, JWT_TOKEN } = process.env;
+const releaseService = {
+  createRelease: async (publicationId: string): Promise<string | null> => {
+    console.time('createRelease');
+    try {
+      const res = await axios({
+        method: 'POST',
+        url: `${ADMIN_URL}/api/publications/${publicationId}/releases`,
+        data: {
+          timePeriodCoverage: { value: 'AY' },
+          releaseName: 2222,
+          typeId: '1821abb8-68b0-431b-9770-0bea65d02ff0',
+          publicationId: 'c76c1bc4-0a01-4116-74cd-08d8eebbf0c1',
+          templateReleaseId: '',
+        },
+        headers: {
+          'Content-Type': 'application/json',
+          Authorization: `Bearer ${JWT_TOKEN}`,
+        },
```

Comment:
Similarly to `publicationService`, we can refactor all of these Axios invocations with a pre-configured Axios instance.